### PR TITLE
PR to merge my routes to API request.

### PR DESCRIPTION
**Pull request by wooseok0717**
My first PR.

complete all routes finally.
please let me know if there are anything I can improve on PRs or code itself.

thank you for your time for reviewing my code!

### CODE REVIEW

File: server/controllers/questions.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+module.exports = {
+  getAllQuestionsByProductId: (req, res) => {
+    // page&count has default value 1&5.
+    // keep this in mind in case you need to change these value.
+    const {page, count, product_id} = req.query;
+    let url = `/qa/questions?page=${page}&count=${count}&product_id=${product_id}`;
+    let config = {
+      method: 'get',
+      baseURL: baseURL,
```
Comment (kevinleaves) [inline server/controllers/questions.js]: I think the airbnb linter would flag this notation. 

Please change baseURL and url keys to use object shorthand.

https://eslint.org/docs/latest/rules/object-shorthand

File: server/controllers/questions.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        res.status(200).send(result.data)
+      })
+      .catch(err => {
+        console.log(err);
+        res.sendStatus(404);
+      })
+  },
+  getAnswersByQuestionId: (req, res) => {
+    // default value for page and counts
+    const {page, count} = req.query;
```
Comment (kevinleaves) [inline server/controllers/questions.js]: Another small linter fix. 

https://eslint.org/docs/latest/rules/object-curly-spacing

**Review by kevinleaves — CHANGES_REQUESTED** [review]
Hey Josh, good work on the routes. 

Very descriptive route handler names.

Only thing I would fix are some slight airbnb linter issues.

File: server/controllers/questions.js (hunk 1 above)
Comment (wooseok0717) [reply server/controllers/questions.js]: Thank you Kev! :)

**Review by kevinleaves — APPROVED** [review]
good stuff. 
looks like your linter works. 
approved.

**Pull request merged**